src/client/Microsoft.Identity.Client/Internal/Logger/LogEntry.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+
+//Will be replaced by logger package when available
+
+namespace Microsoft.IdentityModel.Logging.Abstractions
+{
+    /// <summary>
+    /// Defines the structure of a log entry.
+    /// </summary>
+    public class LogEntry
```

Comment:
Shouldn't a `LogEntry` also have a `IsPii` property?

Reply:
The design of the new identity logger is to assume that messages will contain Pii if the IsPiiEnabled property on the IIdentityLogger is set to true and if it is set to false, messages will not contain Pii. This is different from the old way we used to do it where we would identify if each message contains Pii or not. This is why this class does not have isPii. 

However, after creating this prototype, I noticed that we do need some way to identity whether or not a message contains Pii since the legacy log adapter will need that information for the callback. However, I'm not sure adding a property to the new LogEntry just for our legacy logger callback is the right thing to do?